graph/src/log/mod.rs

```diff
@@ -43,7 +43,7 @@ pub fn logger(show_debug: bool) -> Logger {
                 FilterLevel::Info
             },
         )
-        .parse(ENV_VARS.log_levels())
+        .parse(ENV_VARS.log_levels().unwrap_or_default())
```

Comment:
I guess now I see @leoyvens 's point about `Default`.

Reply:
Good point! Will fix this.